pkg/manifests/manifests.go

```diff
@@ -2723,6 +2736,16 @@ func setArg(args []string, argName string, argValue string) []string {
 	return args
 }
 
+func unsetArg(args []string, argName string) []string {
```

Comment:
I don't think that we need unsetArg (at least for now).

Reply:
ok, unsetArg() will be removed now and added later when we need it.